### PR TITLE
Remove some todos or mark them to fix

### PR DESCRIPTION
**Pull request by NYBACHOK**

### CODE REVIEW

**Review by dmvict — APPROVED** [review]

File: x/ibc-rs/src/ics02_client/params.rs [hunk 1]
```diff
@@ -73,7 +73,7 @@ impl<PSK: ParamsSubspaceKey> ClientParamsKeeper<PSK> {
     ) -> ClientParams {
         let store = infallible_subspace(ctx, &self.params_subspace_key);
 
-        store.params().unwrap() // TODO: Add default
```
Comment (joneskm) [inline x/ibc-rs/src/ics02_client/params.rs:76]: The TODO seemd to be giving some guidance which might be useful

File: x/ibc-rs/src/ics02_client/params.rs [hunk 2]
```diff
@@ -92,7 +92,7 @@ impl<PSK: ParamsSubspaceKey> ClientParamsKeeper<PSK> {
     ) -> Result<ClientParams, GasStoreErrors> {
         let store = gas::subspace(ctx, &self.params_subspace_key);
 
-        Ok(store.params()?.unwrap()) // TODO: Add default
```
Comment (joneskm) [inline x/ibc-rs/src/ics02_client/params.rs:95]: Same as above

File: x/ibc-rs/src/ics03_connection/params.rs [hunk 3]
```diff
@@ -76,7 +76,7 @@ impl<PSK: ParamsSubspaceKey> ConnectionParamsKeeper<PSK> {
     ) -> ConnectionParams {
         let store = infallible_subspace(ctx, &self.params_subspace_key);
 
-        store.params().unwrap() // TODO: Add default
```
Comment (joneskm) [inline x/ibc-rs/src/ics03_connection/params.rs:79]: Same as above

File: x/staking/src/keeper/gov.rs [hunk 4]
```diff
@@ -42,7 +42,7 @@ impl<
         let account = self
             .auth_keeper
             .get_account(ctx, &self.bonded_module.get_address())?
-            .unwrap(); // TODO: Unsure what to do in this case
+            .unwrap(); //  Unsure what to do in this case
```
Comment (joneskm) [inline x/staking/src/keeper/gov.rs:45]: That looks like a TODO?

File: x/staking/src/keeper/mod.rs [hunk 5]
```diff
@@ -263,7 +263,7 @@ impl<
                     .validator(ctx, &last_validator.address)
                     .expect("Init ctx doesn't have any gas")
                     .expect("validator in the store was not found");
-                // TODO: check unwraps and update types to omit conversion
+                // check unwraps and update types to omit conversion
```
Comment (joneskm) [inline x/staking/src/keeper/mod.rs:266]: That looks like a TODO?

File: x/staking/src/keeper/mod.rs [hunk 6]
```diff
@@ -461,7 +461,7 @@ impl<
             if old_power_bytes.is_none()
                 || old_power_bytes.map(|v| v.as_slice()) != Some(&new_power_bytes)
             {
-                // TODO: check unwraps and update types to omit conversion
+                //  check unwraps and update types to omit conversion
```
Comment (joneskm) [inline x/staking/src/keeper/mod.rs:464]: Same as above

File: gaia-rs/src/staking_grpc.rs [hunk 7]
```diff
@@ -1,5 +1,5 @@
 // /!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\
-/// TODO: move to staking module when it's hooked into gaia-rs
+/// TODO:ME move to staking module when it's hooked into gaia-rs
```
[the remark this answers is not in the document]
Comment (NYBACHOK) [reply gaia-rs/src/staking_grpc.rs:2]: Smth that `ME` will fix quickly

File: trees/src/iavl/query_tree.rs [hunk 8]
```diff
@@ -43,7 +43,6 @@ impl<DB: Database> QueryTree<DB> {
         }
     }
 
-    // TODO: can we share this function with a regular tree's get_ method?
```
[the remark this answers is not in the document]
Comment (NYBACHOK) [reply trees/src/iavl/query_tree.rs:46]: No. I don't see any usage for this fn outside, so removed it

File: x/bank/src/keeper.rs [hunk 9]
```diff
@@ -915,7 +915,6 @@ impl<SK: StoreKey, PSK: ParamsSubspaceKey, AK: AuthKeeper<SK, M>, M: Module>
         &self,
         ctx: &mut CTX,
         addr: &AccAddress,
-        // TODO: consider to add struct Coins that can have empty coins list
```
[the remark this answers is not in the document]
Comment (NYBACHOK) [reply x/bank/src/keeper.rs:918]: From my experience, we have no need in this.

File: x/ibc-rs/src/ics02_client/params.rs (hunk 1 above)
Comment (NYBACHOK) [reply x/ibc-rs/src/ics02_client/params.rs:76]: This code needs to removed. Trait impl for params contains blanket impl

File: gears/src/utils.rs [hunk 10]
```diff
@@ -80,7 +80,7 @@ impl TmpChild {
                 "#,
             &vec![],
             &options,
-        )?; // TODO: make it work for windows too?
+        )?; // QA: make it work for windows too? Docker would be answer
```
[the remark this answers is not in the document]
Comment (NYBACHOK) [reply gears/src/utils.rs:83]: Okay. Fun fact, I already used QA

File: gears/src/types/msg/send.rs [hunk 11]
```diff
@@ -66,12 +66,11 @@ impl From<MsgSend> for inner::MsgSend {
 
 impl Protobuf<inner::MsgSend> for MsgSend {}
 
-//TODO: should to Any be implemented at the individual message type?
 impl From<MsgSend> for Any {
     fn from(msg: MsgSend) -> Self {
         Any {
             type_url: "/cosmos.bank.v1beta1.MsgSend".to_string(),
-            value: msg.encode_vec().expect("msg"), // TODO
+            value: msg.encode_vec().expect("msg"),
```
[the remark this answers is not in the document]
Comment (NYBACHOK) [reply gears/src/types/msg/send.rs:73]: I need to replace it with `IBC_ENCODE_UNWRAP` const. It already points such pieces of code

File: gears/src/params/gas/space_mut.rs [hunk 12]
```diff
@@ -44,7 +44,7 @@ impl<DB: Database> GasParamsSpaceMut<'_, DB> {
         Ok(())
     }
 
-    // TODO: dangerous fn as it may break consistency
+    // TODO:ME What about marking it unsafe?
```
[the remark this answers is not in the document]
Comment (NYBACHOK) [reply gears/src/params/gas/space_mut.rs:47]: Okay

File: gears/src/grpc/tx.rs [hunk 13]
```diff
@@ -17,7 +17,7 @@ impl Service for TxService {
         _request: Request<SimulateRequest>,
     ) -> Result<Response<SimulateResponse>, Status> {
         info!("Received a gRPC request tx::simulate");
-        // TODO: run simulation once implemented
+        //  run simulation once implemented
```
[the remark this answers is not in the document]
Comment (NYBACHOK) [reply gears/src/grpc/tx.rs:20]: Okay

File: gears/src/grpc/health.rs [hunk 14]
```diff
@@ -17,23 +17,21 @@ impl HealthService for GearsHealthService {
...
-        //TODO: implement
         unimplemented!()
     }
 
     async fn get_syncing(
         &self,
         _request: Request<GetSyncingRequest>,
     ) -> Result<Response<GetSyncingResponse>, Status> {
-        //TODO: query the node for syncing status and return it
+        // query the node for syncing status and return it
```
[the remark this answers is not in the document]
Comment (NYBACHOK) [reply gears/src/grpc/health.rs:34]: Okay

File: gaia-rs/tests/ibc.rs [hunk 15]
```diff
@@ -94,8 +94,8 @@
 
 //     let cli_args = CliUpdateClient {
 //         client_id: ClientId::from_str("07-tendermint-0")?,
-//         client_message: String::new(), // TODO: !!!!!! what msg here should be?
-//         signer: Signer::from("TODO"),
+//         client_message: String::new(), //  !!!!!! what msg here should be?
+//         signer: Signer::from(""),
```
[the remark this answers is not in the document]
Comment (NYBACHOK) [reply gaia-rs/tests/ibc.rs:98]: For `signer` we need open issue, but for `client_message` better return

[the remark this answers is not in the document]
Comment (NYBACHOK) [reply gaia-rs/tests/ibc.rs:98]: Returned

File: x/staking/src/keeper/gov.rs (hunk 4 above)
Comment (NYBACHOK) [reply x/staking/src/keeper/gov.rs:45]: There is issue to check all `unwraps`

File: gears/src/params/gas/space_mut.rs (hunk 12 above)
[the remark this answers is not in the document]
Comment (NYBACHOK) [reply gears/src/params/gas/space_mut.rs:47]: I returned this comment in separate PR https://github.com/rumos-io/gears/pull/186

File: gears/src/types/msg/send.rs (hunk 11 above)
[the remark this answers is not in the document]
Comment (joneskm) [reply gears/src/types/msg/send.rs]: So it's still a TODO then?

[the remark this answers is not in the document]
Comment (NYBACHOK) [reply gears/src/types/msg/send.rs]: No need to spam todos around the code for this issue. I created dedicated issue

**Pull request closed without merging**